src/components/GiftCard/GiftCard.js

```diff
@@ -19,7 +20,7 @@ const GiftCard = ({title, purchased, id, removeGift}) => {
       {purchased === "purchased" && 
       <p>Purchased</p>}
```

Comment:
I like how you are displaying the purchased as a p tag and then the button if when they want to mark it as purchased!